source/session.ts

```diff
@@ -663,9 +663,9 @@ export class Session<
             "ftrack-user": this.apiUser,
             "ftrack-Clienttoken": this.clientToken,
             "ftrack-pushtoken": pushToken,
-            "ftrack-api-options": ensureSerializableResponse
-              ? "strict:1;denormalize:1"
-              : undefined,
+            ...(ensureSerializableResponse && {
```

Comment:
This works.

Tiny nit code structure things:
I'd prefer a ternary to `&&`, I had to double check that spreading false (That this would evaluate to if ensureSerializableResponse) would be the same as spreading an empty object.

(Note that this is a personal preference, if you don't agree, don't change it)

Reply:
made to ternary as it is more readable. 
Added test case.